flow/connectors/postgres/validate.go

```diff
@@ -130,3 +137,35 @@ func (c *PostgresConnector) CheckReplicationConnectivity(ctx context.Context) er
 
 	return conn.Close(ctx)
 }
+
+// Check if all tables have either a primary key or REPLICA IDENTITY FULL
+func (c *PostgresConnector) CheckIfTablesAreMirrorable(
+	ctx context.Context,
+	tables []*utils.SchemaTable,
+) error {
+	badTables := make([]string, 0, len(tables))
```

Comment:
```suggestion
	var badTables []string
```
This doesn't need to allocate anything except in error path